src/flake8/utils.py

```diff
@@ -56,7 +56,7 @@ class _Token(NamedTuple):
 _CODE, _FILE, _COLON, _COMMA, _WS = "code", "file", "colon", "comma", "ws"
 _EOF = "eof"
 _FILE_LIST_TOKEN_TYPES = [
-    (re.compile(r"[A-Z]+[0-9]*(?=$|\s|,)"), _CODE),
+    (re.compile(r"[A-Z]+[A-Z0-9-]*(?=$|\s|,)"), _CODE),
```

Comment:
This may work but it's the wrong change to make.

_If_ we were to accept this, which I don't think we will, the regular expression should be significantly different, something more like:

```pcre
[A-Z][A-Z-]+[0-9]*(?=$|\s|,)
```

Reply:
Sure completely open to adjusting the regex. But I'm unsure of the need to be so restrictive (numbers can only be at the end for example). I've written up a little more in the feature request "issue".